handler/src/main/java/io/netty/handler/ssl/SslContextBuilder.java

```diff
@@ -573,11 +588,12 @@ public SslContext build() throws SSLException {
             return SslContext.newServerContextInternal(provider, sslContextProvider, trustCertCollection,
                 trustManagerFactory, keyCertChain, key, keyPassword, keyManagerFactory,
                 ciphers, cipherFilter, apn, sessionCacheSize, sessionTimeout, clientAuth, protocols, startTls,
-                enableOcsp, keyStoreType);
+                enableOcsp, keyStoreType, toArray(options.entrySet(), new Map.Entry[0]));
         } else {
             return SslContext.newClientContextInternal(provider, sslContextProvider, trustCertCollection,
                 trustManagerFactory, keyCertChain, key, keyPassword, keyManagerFactory,
-                ciphers, cipherFilter, apn, protocols, sessionCacheSize, sessionTimeout, enableOcsp, keyStoreType);
+                ciphers, cipherFilter, apn, protocols, sessionCacheSize, sessionTimeout, enableOcsp, keyStoreType,
+                    toArray(options.entrySet(), new Map.Entry[0]));
```

Comment:
@chrisvest the reason is that I want to ensure we actually can't affect the configured options once build is called. Otherwise it may be possible in the future that some of these options are used in a lazy fashion and a user may call `.option(...)` after `.build()` and so change the already built `SslContext`.

Reply:
Also I thought not using varargs makes things more explicit.